pkg/cli/admin/inspect/pod.go

```diff
@@ -290,6 +293,17 @@ func (o *InspectOptions) gatherContainerLogs(destDir string, pod *corev1.Pod, co
 			Previous:   false,
 			Timestamps: true,
 		}
+		if o.since != 0 {
+			sec := (int64(o.since.Round(time.Second).Seconds()))
+			logOptions.SinceSeconds = &sec
+		}
+		if len(o.sinceTime) > 0 {
+			t, err := util.ParseRFC3339(o.sinceTime, metav1.Now)
```

Comment:
I'd expect this to fail the command's validation phase, iow. move this to inspect.go's `Complete` method and add another member in `InspectOptions` which will hold already parsed time value.